docs/deprecated.md

```diff
@@ -1063,75 +1070,78 @@ Since 1.9, Docker Content Trust Offline key has been renamed to Root key and the
 
 The endpoint `/containers/(id or name)/copy` is deprecated in favor of `/containers/(id or name)/archive`.
 
-
 ### LXC built-in exec driver
+
 **Deprecated In Release: [v1.8.0](https://github.com/docker/docker/releases/tag/v1.8.0)**
 
 **Removed In Release: [v1.10.0](https://github.com/docker/docker/releases/tag/v1.10.0)**
 
 The built-in LXC execution driver, the lxc-conf flag, and API fields have been removed.
 
-
 ### Old Command Line Options
+
 **Deprecated In Release: [v1.8.0](https://github.com/docker/docker/releases/tag/v1.8.0)**
 
 **Removed In Release: [v1.10.0](https://github.com/docker/docker/releases/tag/v1.10.0)**
 
 The flags `-d` and `--daemon` are deprecated in favor of the `daemon` subcommand:
 
-    docker daemon -H ...
+```console
+docker daemon -H ...
+```
```

Comment:
The `docker daemon` subcommand is now also removed (because it's a separate `dockerd` binary now), so not sure we should show this as example 🙈 https://github.com/docker/cli/blob/22300893711db8ebc6bb1bd827ca1933392e8b60/docs/deprecated.md#docker-daemon-subcommand